styles.css

```diff
@@ -1,3 +1,3 @@
 .color_text {
-    color: aquamarine;
+  color: aquamarine;
 }
```

Comment:
[Required] it would be better if you choose a better title for your pull request @ABDULALI3468 🚀

Pull Request should have a descriptive title telling you what changes are inside.

![image](https://user-images.githubusercontent.com/74744787/158466774-ed8e08b2-4556-4964-b026-3635835cce44.png)

choose a title that describes the changes that you did  to your project example : 
hello_microverse

the reason for this is to give a better understanding of the changes you did and a better documentation